ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/PubsubMessageToTableRow.java

```diff
@@ -164,37 +163,6 @@ private PubsubMessageToTableRow(ValueProvider<String> tableSpecTemplate) {
     return result;
   }
 
-  @VisibleForTesting
-  static TableRow buildTableRow(byte[] payload) throws IOException {
-    TableRow tableRow = Json.readTableRow(payload);
-    promoteFields(tableRow);
-    return tableRow;
-  }
-
-  /**
-   * BigQuery cannot partition or cluster tables by a nested field, so we promote some important
-   * attributes out of metadata to top-level fields.
-   */
-  private static void promoteFields(TableRow tableRow) {
```

Comment:
promoteFields is no longer needed, because AddMetadata is now responsible for adding top-level fields like submission_date.